tox.ini

```
@@ -0,0 +1,23 @@
+[flake8]
+ignore=
+    ; module level import not at top of file
+    E402,
+    ; line break before binary operator
+    W503,
+    ; line break after binary operator
+    W504
+    ; "experimental" SIM9xx rules (flake8-simplify)
+    SIM9
+    ; suggests using f"{!r}" instead of manual quotes (flake8-bugbear)
+    ; Doesn't work at 3.7
+    B028
+
+exclude=
+    build,
+    dist,
+    .git,
+    __pycache__,
+    .tox,
+paths =
+    ./cylc/flow
```

I think this turns linting off:

```suggestion
    ./cylc/rose
```

(please test that linting still works after this change with no extra ages though!)